### ISSUE TITLE
Problems with Python 2.7.12

### ISSUE THREAD

**Issue by marianobianchi** [neurobin/letsacme#4]
Hello, I was trying to use your script in my webfaction account. When I ran the command

```
python letsacme.py --no-chain --account-key account.key --csr domain.csr --acme-dir challenges > signed.crt
```

I got the error

> ssl.CertificateError: hostname 'USERNAME.webfactional.com' doesn't match either of '*.webfaction.com', 'webfaction.com'

I think it is an error with SNI. When I ran it using Python 3.3.6 it worked like a charm

**Comment by neurobin** [comment]
Are you using wildcard domain in CSR?

**Comment by marianobianchi** [comment]
No. I was trying to use it for my webfaction account. It is USERNAME.webfactional.com
